prime-angular-frontend/src/app/modules/adjudication/shared/components/review-status-content/review-status-content.component.ts

```diff
@@ -126,7 +139,11 @@ export class ReviewStatusContentComponent implements OnInit, OnChanges {
           return reasons.concat(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statusReasons'), esr.reasonNote, this.enrollee.identificationDocuments));
         }
 
-        reasons.push(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statusReasons'), esr.reasonNote));
+
+        reasons.push((esr.statusReasonCode === EnrolmentStatusReasonEnum.POSSIBLE_PAPER_ENROLMENT_MATCH)
+          ? this.parsePotentialMatchIds(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statusReasons'), esr.reasonNote))
```

Comment:
You name this correctly as `parsePotentialMatchIds` so the key should be something similar like `potentialMatchIds`

Reply:
renamed